### ISSUE TITLE
RobotX 2024: Task 5 - Scan the Code TEST CASES

### ISSUE THREAD

**Issue by DaniParr** [uf-mil/mil#1264]
### What needs to change?

Write test cases for [Task 5](https://github.com/uf-mil/mil/issues/1256). Create these by modifying existing worlds and saving those world files to be deployed on ROS launch as an argument.

### How would this task be tested?

1. Launch ROS: `roslaunch navigator_launch simulation.launch world:=<name_of_world> --screen`

**Comment by DaniParr** [comment]
@hoshbosh @edwardguthrie Hey y'all any progress on this issue?

**Comment by hoshbosh** [comment]
@DaniParr Yeah I was able to make some worlds with the display in different places relative to the boat, but I was having some trouble getting it pushed. I was planning on coming into the lab to get that sorted out.

**Comment by DaniParr** [comment]
@hoshbosh Sounds good. I will be in the lab during my office hours and this week Wednesday from 1:00 -> 4:00 PM.
